hritam-mondal/TestTrek
# Add authentication and authorization feature

# Add Authentication and Authorization Feature

## Description
This pull request introduces the authentication and authorization feature to the application.

### Screenshots
![image](https://github.com/user-attachments/assets/ac2e2694-2738-40bf-9c25-e7cc847ab800)

## How to Test
Outline the steps to test the changes made in this pull request:
1. Clone the repository: `git clone git@github.com:hritam-mondal/TestTrek.git`
2. Checkout to the feature branch: `git checkout feature/auth`
3. Set up the environment and configuration variables as outlined in the README
4. Run the application: `dotnet run`
5. Verify that user endpoints are working as expected
